blocks/carousel/carousel.css

```diff
@@ -19,6 +19,8 @@ Modifications (preferably avoid) may have been made to fit the specific needs of
     scroll-behavior: smooth;
     scroll-snap-type: x mandatory;
     overflow: scroll clip;
+
+    --visible-slides: 1;
```

Comment:
@sneharora1  is this expected to be reuse somewhere  apart from one place in this PR.
and this is being defined in 
`.carousel .carousel-slides {`
what would be the scope of this variable ?

Reply:
Its used in line#35. This is coming from @shrotia's PR.